virttest/libvirt_vm.py

```diff
@@ -370,6 +370,12 @@ def has_option(help_text, option):
         def has_os_variant(os_text, os_variant):
             return bool(re.search(r"%s" % os_variant, os_text, re.MULTILINE))
 
+        def has_sub_option(option, sub_option):
+            option_help_text = process.system_output("%s --%s help" %
+                                                     (virt_install_binary, option),
+                                                     verbose=False)
+            return bool(re.search(r"%s" % sub_option, option_help_text, re.MULTILINE))
```

Comment:
@will-Do Other helper functions for command line sits there, that's the reason have placed it,
not finding any other appropriate place right now, Hope it should be fine?